services/fluent-bit/0.20.5/resource-quota/resourcequota.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: fluent-bit-critical-pods
+  namespace: ${releaseNamespace}
+spec:
+  hard:
+    pods: "1G"
```

Comment:
limit of pods in that priority class, yes. The number was taken from the resource quota that GKE deploys to the `kube-system` namespace

Reply:
gotcha, in the future we may need this to be configurable but it seems good for now!

Reply:
yeah def, I noted the concern to craig but he's happy going with this change with that caveat
> I think as long as we don't make it too hard for a savvy customer to adjust the FB filters and settings, we're good, as I've seen people asking about that before.    I think it would be a rare case for anyone to want to adjust/change the priority class/quota.